src/integrationTest/resources/config.yml

```diff
@@ -16,5 +16,5 @@ config:
           type: "internal"
           config: {}
     on_behalf_of:
-      signing_key: "signing key"
-      encryption_key: "encryption key"
+      signing_key: "VGhpcyBpcyB0aGUgand0IHNpZ25pbmcga2V5IGZvciBhbiBvbiBiZWhhbGYgb2YgdG9rZW4gYXV0aGVudGljYXRpb24gYmFja2VuZCBmb3IgdGVzdGluZyBvZiBleHRlbnNpb25z"
```

Comment:
NIT: Add a comment for what these are--how did you get them and how can we update them etc.

Reply:
Good idea I can do a comment above to show the plain text version of them.

Reply:
Done